### PR TITLE
improve csv separator notes

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -370,7 +370,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --cert | 用于 TLS 连接的 client certificate 文件的地址 |
 | --key | 用于 TLS 连接的 client private key 文件的地址 |
 | --csv-delimiter | csv 文件中字符类型变量的定界符 | '"' |
-| --csv-separator | csv 文件中各值的分隔符 | ',' |
+| --csv-separator | csv 文件中各值的分隔符，不建议使用默认的“,”简单分隔符，推荐“|+|”等非常见字符组合| ','|
```
Comment (hfxsd) [inline dumpling-overview.md:373]: 应该在产品侧进行改进。

**Review by hfxsd — APPROVED** [review]
LGTM

File: dumpling-overview.md (hunk 1 above)
Comment (hfxsd) [inline dumpling-overview.md:373]:
```suggestion
| --csv-separator | csv 文件中各值的分隔符，不建议使用默认的“,”简单分隔符，推荐“\|+\|”等非常见字符组合| ','|
```

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by Liuxiaozhen12 — APPROVED** [review]

File: dumpling-overview.md [hunk 2]
```diff
@@ -370,7 +370,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --cert | 用于 TLS 连接的 client certificate 文件的地址 |
 | --key | 用于 TLS 连接的 client private key 文件的地址 |
 | --csv-delimiter | csv 文件中字符类型变量的定界符 | '"' |
-| --csv-separator | csv 文件中各值的分隔符 | ',' |
+| --csv-separator | csv 文件中各值的分隔符，不建议使用默认的','简单分隔符，推荐非常见字符组合| ','|
```
Comment (sunzhaoyang) [inline dumpling-overview.md:373]:
```suggestion
| --csv-separator | csv 文件中各值的分隔符，如果数据中可能有逗号（二进制类型、字符串类型），建议分隔符使用非常见组合字符| ','|
```

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ CSV 格式可在 `tidb-lightning.toml` 文件中 `[mydumper.csv]` 下配置。
 
 ```toml
 [mydumper.csv]
-# 字段分隔符，必须为 ASCII 字符。
+# 字段分隔符，必须为 ASCII 字符。不建议源文件使用默认的','简单分隔符，推荐'|+|'等非常见字符组合
```
Comment (sunzhaoyang) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:31]:
```suggestion
# 字段分隔符，必须为 ASCII 字符。如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符例如'|+|'。
```

File: dumpling-overview.md (hunk 2 above)
Comment (sunzhaoyang) [inline dumpling-overview.md:373]:
```suggestion
| --csv-separator | csv 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
```

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 4]
```diff
@@ -161,7 +161,7 @@ strict-format = true
 
 ```toml
 [mydumper.csv]
-separator = ','
+separator = ',' # 不建议源文件使用默认的','简单分隔符，推荐'|+|'等非常见字符组合
```
Comment (sunzhaoyang) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:164]:
```suggestion
separator = ',' # 如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符例如'|+|'
```

**Review by lance6716 — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Pull request merged**